apps/frontend/src/common/containers/insightConsentForm/InsightConsentForm.module.scss

```diff
@@ -0,0 +1,11 @@
+.wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.wrapper p + h1,
+.wrapper p + div,
+.wrapper p + p,
+.wrapper div + h1 {
```

Comment:
What goes on here? I am very confuse

Reply:
Haha, this is for managing margins above each header and paragraph, but not between paragraphs and headers. 

Further reading if interested: https://www.w3schools.com/cssref/sel_element_pluss.asp